### PR TITLE
Update README and CONTRIBUTING; create STYLE

### PR DESCRIPTION
**Pull request by djmagee**
## What does this PR do?

- Update README.md: Incorporate information from old About page.
- Update CONTRIBUTING.md: Remove most information in favor of requesting GitHub issues. Move style guidance to separate file.
- Create STYLE.md: Incorporate and update style guidance from CONTRIBUTING.md.
- Create LICENSE.md: Add MIT License, currently in old About page; remove existing LICENSE.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,83 +1,8 @@
...
-
-### Capitalize core terms
-
-Many of Temporal's core terms can be used in a generic way.
-To differentiate between one of Temporal's core terms vs a generic instance of a term, always treat Temporal terms as proper nouns in documentation.
-Generic versions of the same term should not be capitalized and should be used sparingly to avoid confusion.
-
-- Correct: "... register the Activity within the Workflow... "
-- Incorrect: "... register the activity within the workflow..."
+If you are outside the Temporal organization, please create a [GitHub issue](https://github.com/temporalio/documentation/issues) for any feedback more significant than a simple typo.
```
Comment (flossypurse) [inline CONTRIBUTING.md:7]: Let's put this first.
1 Create a GitHub Issue (for anything more than typos and small fixes)
2 If you end up putting up a change, then follow style guidance

File: README.md [hunk 2]
```diff
@@ -1,10 +1,42 @@
...
 
-Please read the [CONTRIBUTING](/CONTRIBUTING.md) guide before you submit any pull requests.
+Please read the [CONTRIBUTING](/CONTRIBUTING.md) guide and the [STYLE](/STYLE.nd) guide before you submit any pull requests.
 
 Maintainers and contributors to this project are expected to conduct themselves in a respectful way.
 See the [CNCF Community Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) as a reference.
+
+The source of this documentation is available to the public for individual and commercial use.
+
+MIT License
```
Comment (flossypurse) [inline README.md:18]: We might want to put this as a stand alone file and then link to it.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,83 +1,8 @@
...
+We encourage contributions from the community.
+When creating content, please follow our [style guidance](/STYLE.md).
 
 To ensure that each change is relevant and properly peer reviewed, we request that you adhere to best practices for open-source contributions.
-This means that if you are outside the Temporal organization, you must fork the repository and create pull requests from branches on your own fork.
-GitHub's [first-contributions repo README](https://github.com/firstcontributions/first-contributions) provides an example of how to do that.
-
-When it comes to crafting content, please follow our [style guidance](#style-guidance).
-
-## Preview changes locally
```
Comment (flossypurse) [inline CONTRIBUTING.md:11]: I think we should still keep the "How to install and run this site locally" info in the README or CONTRIBUTING - but within context

File: CONTRIBUTING.md (hunk 1 above)
Comment (djmagee) [reply CONTRIBUTING.md:7]: Done.

File: README.md (hunk 2 above)
Comment (djmagee) [reply README.md:18]: Done. I added LICENSE.md to this PR.

File: CONTRIBUTING.md (hunk 3 above)
Comment (djmagee) [reply CONTRIBUTING.md:11]: Right-o.

**Review by aarohib — APPROVED** [review]

**Review by flossypurse — APPROVED** [review]
Awesome.
Looks like we have 2 LICENSE files now though - we probably only need 1.

**Comment by djmagee** [comment]
_D'oh._ Totally missed the existing LICENSE file. I deleted it in favor of LICENSE.md.

**Pull request merged**